From f6fee1399208939d7975a166a96c9bf246d7b233 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:25:32 -0400
Subject: [redhat] kabi: Adding symbol submit_bio (block/blk-core.c)

Message-id: <66a44de97914afffd4adca61701ff116ba406d66.1410510773.git.poros@redhat.com>
Patchwork-id: 96055
O-Subject: [PATCH RHEL7.1 212/233] kabi: Adding symbol submit_bio (block/blk-core.c)
Bugzilla: 1071099
RH-Acked-by: Jeff Moyer <jmoyer@redhat.com>

Bugzilla: 1071099
    https://bugzilla.redhat.com/show_bug.cgi?id=1071099

    Description:
    ============
    Submit a bio to the block device layer for I/O.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/submit_bio
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/submit_bio
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/submit_bio

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/submit_bio b/redhat/kabi/kabi-module/kabi_ppc64/submit_bio
new file mode 100644
index 0000000..fe1f022
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/submit_bio
@@ -0,0 +1,2 @@
+#1-
+0x845cfa7e submit_bio vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/submit_bio b/redhat/kabi/kabi-module/kabi_s390x/submit_bio
new file mode 100644
index 0000000..608311a
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/submit_bio
@@ -0,0 +1,2 @@
+#1-
+0xc871bd8e submit_bio vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/submit_bio b/redhat/kabi/kabi-module/kabi_x86_64/submit_bio
new file mode 100644
index 0000000..1fae41d
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/submit_bio
@@ -0,0 +1,2 @@
+#1-
+0xc4076f47 submit_bio vmlinux EXPORT_SYMBOL
-- 
1.7.1